test/functional/test-cid-api.js

```diff
@@ -248,7 +248,7 @@ describes.realWin('test-cid-api', {}, env => {
   it('should return null if AMP_TOKEN=$OPT_OUT ', () => {
     persistCookie('AMP_TOKEN', '$OPT_OUT');
     return api.getScopedCid('googleanalytics', 'scope-a').then(cid => {
-      expect(cid).to.be.null;
+      expect(cid).to.equal('$OPT_OUT');
```

Comment:
a unit test exists: https://github.com/lannka/amphtml/blob/b75e04837cb44918a509285ef131e925252cce40/test/functional/test-cid.js#L817

also tested manually by changing the browser cookie to $OPT_OUT